Makefile

```diff
@@ -1,6 +1,6 @@
 DEBIAN_VERSION ?= stretch
 
-DEBIAN_VENTI_GOVERSIONS ?= 1.8 1.9 1.10.3
+DEBIAN_VENTI_GOVERSIONS ?= 1.8 1.9 1.10.3 1.10.7 1.11.4
```

Comment:
Replace `1.10.3` with `1.10.7` instead of adding?

Reply:
These images are used fairly widely across our code base, I'm not sure I can guarentee I'll find and update all the references across all our branches. And then the item will be pulling an out of date image from quay. If I delete the old tag on quay, that would force any builds to fail and need to be updated to the latest version, but could be problematic for any of those lurking issues. 

So I don't know what's best, by being additive I opted for the route I think is the least likely to cause us a bunch of headaches.

Reply:
Sounds fair.

Reply:
To be honest I still see a lot of references to tall/vendi/grande:0.0.1 which using Debian jessie, what is a bigger problem than out of date golang image.

Reply:
I agree, this should all be updated as well, and possibly move to ubuntu or scratch containers for some items. But I think this is separate from trying to tackle this specific CVE.